docs/user/03-10-preconfigured-secret.md

```diff
@@ -43,12 +43,14 @@ The cluster ID represents a Kyma service instance created in a particular subacc
 
 ## Cluster Access
 
-By default, SAP BTP Operator has cluster-wide permissions. You cannot reconfigure the predefined settings.
+By default, SAP BTP Operator has cluster-wide permissions.
 
 The following parameters manage cluster access:
 
 | Parameter                     | Description                                                                                   |
 |-------------------------------|-----------------------------------------------------------------------------------------------|
 | **CLUSTER_ID**                | Generated when Kyma runtime is created.                                                       |
-| **MANAGEMENT_NAMESPACE**      | Always set to `kyma-system`.                                                |
+| **MANAGEMENT_NAMESPACE**      | By default, set to `kyma-system`.                                                |
```

Comment:
I am not sure that this config does and I cannot find it in out documentation. Maybe we could clarify it at least as a followup?

Reply:
I'm not sure what to clarify here. Can you please clarify? This section is based on [this source](https://github.com/SAP/sap-btp-service-operator/blob/main/README.md#managing-access) and modified according to PK's input.

Reply:
More info has been added.